example/mxnet_adversarial_vae/README.md

```diff
@@ -1,22 +1,55 @@
-VAE-GAN in MXNet
+# VAE-GAN in MXNet
 
-Implementation of Autoencoding beyond pixels using a learned similarity metric based on the Tensorflow implementation of https://github.com/JeremyCCHsu/tf-vaegan/
```

Comment:
Can you hyperlink "Autoencoding beyond pixels using a learned similarity metric" to the paper: https://arxiv.org/abs/1512.09300

Reply:
New sentence for: "Based on the Tensorflow implementation ..."